cmake/ElektraConfig.cmake.in

```diff
@@ -2,32 +2,29 @@
 #
 # Variables for the Elektra library:
 #
-#  Elektra_INCLUDE_DIRS  - where to find kdb.h, etc.
-#  Elektra_LIBRARIES     - List of libraries when using Elektra.
+# Elektra_INCLUDE_DIRS  - where to find kdb.h, etc. Elektra_LIBRARIES     - List of libraries when using Elektra.
```

Comment:
This formatting got broken.

@sanssecours any idea how to avoid this?

Reply:
The easiest option is probably using a fence:

```cmake
# ~~~
```

before and after the comment block, just like we did here:

https://github.com/ElektraInitiative/libelektra/blob/bc9254cd9ab37c10275f8e63763d14ee3560bec5/cmake/ElektraCache.cmake#L1-L9

.